addons/wp/functions/fnc_doArtillery.sqf

```diff
@@ -163,7 +169,9 @@ if (_markerList isNotEqualTo []) then {
 sleep _checkRounds;
 
 // re-add to list
-if (!canFire _gun) exitWith {false};
+if (!canFire _gun) exitWith {
+    [QGVAR(RegisterArtillery), [_gun]] call CBA_fnc_serverEvent;
+    false
+};
 
 // ready up again
-[QGVAR(RegisterArtillery), [_gun]] call CBA_fnc_serverEvent;
```

Comment:
Why remove that? Artillery gun will then not be added if they successfully fired and are ready to fire more?

Reply:
Because I am stupid xD will revert that